shinysdr/i/db.py

```diff
@@ -186,11 +186,11 @@ def __init__(self, db, instantiate):
         self.__instantiate = instantiate
     
     def render_GET(self, request):
-        request.setHeader(b'Content-Type', b'application/json')
+        request.setHeader(b'Content-Type', b'application/json; charset=utf-8')
```

Comment:
application/json does not have a charset parameter; this change and its other instances below should be reverted. https://tools.ietf.org/html/rfc7159

Reply:
I get what the RFC says, but I believe this was in fact necessary to make it work because something else started setting/requiring it. I'll try to reproduce the error so I can post it here.

(I certainly didn't spontaneously decide to add this all over the place!)

Reply:
I can't reproduce whatever error I saw (and this was months ago so I've long since forgotten). I reverted these changes for now as a separate commit so I can easily re-add them if/when I figure out why they were necessary in the first place.